CONTRIBUTING.md

```diff
@@ -1,11 +1,14 @@
-Contributing
-============
-If you want to contribute to LAYNII and make it better, your help is very welcome.
+# Contributing
```

Comment:
I think a `ctrl+s` did autoreformat to enfore hard word wrapping. 

No actual content change in CONTRIBUTING.md except for the part related to docker

Sorry for the extra noise in the PR.

Reply:
Looks fine, thanks for the clarification.